ci/scale-pure-ovn-hosts.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Save trace setting
+XTRACE=$(set +o | grep xtrace)
+set -o xtrace
+
+# Read variables
+source ovn-scale.conf
```

Comment:
Eventually, but now it doesn't run any tests, so lets wait until I do that. This is the first step. :)